doc/api/inspector.md

```diff
@@ -0,0 +1,80 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: REPLACEME
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
```

Comment:
I mentioned EventEmitter and also split the events section to make it clearer that there is a generic notification event as well as specific events for protocol notifications.